ember/.stylelintrc.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+module.exports = {
+  extends: ["stylelint-config-standard-scss", "stylelint-prettier/recommended"],
+  rules: {
+    "scss/at-extend-no-missing-placeholder": null,
+  },
```

Comment:
Why not disabling it inline as proposed? Those rules are our friends. :upside_down_face: